### ISSUE TITLE
Fsr Mod dosent work after update.

### ISSUE THREAD

**Issue by ILyesMk2** [NarutoUA/gta5_fsr#95]
dosent work after the San Andreas Mercenaries update. (1.67)
i re installed it and used debug view but nothing happens it used to work fine before the update and ive been using it for along time

**Comment by NarutoUA** [comment]
Looks like devs updated anticheat and d3d11.dll is now loaded from system folder directly. I guess it also broke enb and similar mods

**Comment by ILyesMk2** [comment]
Yes! exactly all other mods that use their own d3d11.dll are not working anymore. including Reshade / enb / this mod. trust me i tried all of them.
just like they did with dxvk 4 years ago 

(edit): reshade still works i accidentally wrote it lol. it uses dxgi anyway

**Comment by ILyesMk2** [comment]
just like they did with dxvk 4 years ago

**Comment by Snipy44** [comment]
but fivem launch the dll in the plugin folder. but ENB proxy dont work

**Comment by KYPY3HO** [comment]
Guys, i think i found solution. You need to go to the Gta 5 installation folder and enable compatibility mode with windows 7 in the properties of absolutely all exe files, i check it and it is work. Source: https://www.youtube.com/watch?v=ESWkbEM0xsY&ab_channel=BERKLEY
